schema/migration-2-0002-20201026.sql

```diff
@@ -0,0 +1,29 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 2 THEN
+    EXECUTE 'ALTER TABLE "block" ADD COLUMN "invalid_before" uinteger NULL' ;
+    EXECUTE 'CREATe TABLE "ma_policy"("id" SERIAL8  PRIMARY KEY UNIQUE,"hash" hash28type NOT NULL)' ;
```

Comment:
Is the casing for `CREATe` intentional?

Reply:
Its generated that way by Haskell's Persistent library.